### DISCUSSION TITLE
how long to check the PeriodicTask model

### DISCUSSION THREAD

**Discussion by duyuyouci** [celery/django-celery-beat#491] [Q&A]
### Summary:

Include a *brief* descrioption of the problem here, and fill out the version info below.  

* Celery Version: 4.4.7
* Celery-Beat Version: 1.5.0

I found that after the timing task is executed, enable of the PeriodicTask will be set to false after a period of time
i want to know how long to check the PeriodicTask

**Comment by auvipy** [comment]
please try latest release, those versions are not supported anymore.
